app/utils/serializer.scala

```diff
@@ -84,6 +84,25 @@ package object serializer {
 
   }
 
+  /**
+    * Banner serializer.
+    */
+  class BannerFormat(implicit routesResolver: RoutesResolver) extends Format[Banner] {
+
+    override def reads(json: JsValue): JsResult[Banner] = JsSuccess(Banner(json.as[Option[String]], None))
+
+    override def writes(a: Banner): JsValue = {
```

Comment:
Try to avoid single letter variable names if their usage is spread over more than two or three consecutive lines.